docs/backup-restore.md

```diff
@@ -0,0 +1,107 @@
+
+### Cassandra
+
+Since Cassandra has a built-in replication mechanism, we don't always need a transactionally-consistent backup to recover a Cassandra cluster.
+
+For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, we don't need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
+However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
+
+To minimize mistakes when doing backup operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+
+Cassy is integrated with `scalar-admin`, so it pauses the scalar DL service to create a transactionally-consistent cluster-wide backup.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### JDBC
+
+You can take a backup with your favorite way for JDBC databases.
+One requirement for backup in Scalar DL on JDBC databases is that backups for all the Scalar DL managed tables (including the coordinator table) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
+That means that you need to create a consistent snapshot by dumping all tables in a single transaction. For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
+
+### Cosmos DB
+
+You should create Cosmos DB account using `Continuous` backup policy to create continuous backups.
+
+Follow [How to create transactionally consistent backup](#how-to-create-transactionally-consistent-backup) to create a transactionally-consistent backup.
+
+### DynamoDB
+
+You should create the schema using the scalardl schema loader, which enables Point-in-time recovery (PITR) for each table in a DynamoDB by default.
+
+Follow [How to create transactionally consistent backup](#how-to-create-transactionally-consistent-backup) to create a transactionally-consistent backup.
+
+## Restore
+
+This section shows how to restore transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must use the midtime of the pause as a restore point.
+* You must restore Scalar Ledger and Auditor tables with the same restore point if you use Ledger and Auditor.
+
+### Cassandra
+
+To minimize mistakes when doing restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+
+### JDBC
+
+You can restore the backup with your favorite way for JDBC databases.
+you can use `mysql` command in MySQL and `psql` command in PostgreSQL to achieve that. Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, 
```

Comment:
Change 'you' to 'You' since it is a sentence start.

Reply:
Thank you for your review comments.
Document updated 6c3bee7